oasis/my-account.md

```diff
@@ -7,51 +7,51 @@ redirect_from:
 ---
 # My Account
 
-You can access information related to your account via the _User Toolbar_.
+You can access information related to your account via the __User Toolbar__.
 The toolbar is in the top right corner in the Oasis dashboard and
 accessible from every view. There are three elements:
 
-- _Bell icon_: Notifications ([Invites](#invites))
-- _Question mark icon_: Help
-- _User icon_: My Account
+- __Bell icon__: Notifications ([Invites](#invites))
```

Comment:
The updated screenshot doesn't have the bell icon anymore, but the text still refers to it. Please fix it.

Reply:
I messed something up and the commit ended up in the other PR: https://github.com/arangodb/docs/pull/914/commits/ffc4c2bf920c4f8c55c687ce360e32f8676dd9be